ci/do_ci.sh

```diff
@@ -463,11 +463,12 @@ case $CI_TARGET in
             --define log_debug_assert_in_release=disabled
         echo "Building binary with wasm=wasmtime... and logging disabled"
         bazel build "${BAZEL_BUILD_OPTIONS[@]}" \
+            --action_env=TEST=TEST \
```

Comment:
for any timed test runs `TEST=TEST` will need to be changed to eg `TEST=TEST23`

Reply:
seems this messed up the ICU build - was wondering whether setting `TEST` would be safe - probs make it something like `ARBITRARY_TEST`

Reply:
emm...yea, TEST may be used by something.